### ISSUE TITLE
No way to see alt text on full-size images that are loading or fail to load

### ISSUE THREAD

**Issue by kindfulkirby** [mastodon/mastodon#38894]
### Steps to reproduce the problem

1. Have an instance with slow or broken image caching/Throttle networks speed in browser dev tools.
2. Click on a picture attached to a post.

### Expected behaviour

To quote Wikipedia: "alternative text (alt text) [that] is to be displayed in place of an element that cannot be rendered"

### Actual behaviour

A colorful gradient with no text appears, with no way to make text appear.

### Detailed description

Alt text should be displayed when an image is loading or fails to load. It is not.

### Mastodon instance

mastodon.social

### Mastodon version

v4.6.0-nightly.2026-05-01

### Browser name and version

Firefox 149.0.2

### Operating system

Fedora Linux 43
